### PR TITLE
Implemented functions to get current url and current path using session

### PR DESCRIPTION
**Pull request by praveenperera**

### CODE REVIEW

File: lib/wallaby/session.ex [hunk 1]
```diff
@@ -124,6 +124,22 @@ defmodule Wallaby.Session do
...
+  """
+  @spec get_current_url(t) :: String.t
+  def get_current_url(session) do
+     Driver.current_url(session)
+  end
+
+  @doc """
+  Gets the current url of the session.
+  """
+  @spec get_current_url(t) :: String.t
```
Comment (keathley) [inline lib/wallaby/session.ex]: Can you update this to the correct spect? I think it should be `get_current_path`. Looks like the `@doc` needs to be updated as well.

**Comment by keathley** [comment]
Thanks for this! I left a few comments. Just some simple updates to the function spec and docs. Otherwise it looks good to me.

**Comment by praveenperera** [comment]
Oops my mistake with the typos, fixed.

**Comment by keathley** [comment]
Looks great!

**Pull request merged**